templates/resources/environment.md.tmpl

```diff
@@ -27,6 +27,9 @@ resource "axual_environment" "development" {
   authorization_issuer = "Auto"
   instance = data.axual_instance.testInstance.id
   owners = axual_group.tenant_admin_group.id
+   settings = {
+      enforceDataMasking = true
```

Comment:
Can we please document what env settings are available? Or at least link to axual docs where this information is available?

Reply:
@Farnaz-java-dev No, we do not have documentation for this yet.

Reply:
ok, I will mention the existing setting in the doc